extra/release_tool.py

```diff
@@ -418,12 +419,15 @@ def version_of(integration_dir, yml_component, in_integration_version=None):
             if len(rev_range) > 1:
                 range_type = ".."
         repo_range = []
+
+        remote = ""
+        match = re.match("^(.*)/", rev_range[0])
+        if match is not None:
+            remote_candidate = match.group(1)
+            if subprocess.call("git remote  | egrep -q ^%s$" % remote_candidate, shell=True) == 0:
+                remote = remote_candidate + "/"
```

Comment:
I have a slightly cleaner approach:

```bash
remote_candidate = match.group(1)
if subprocess.call("git rev-parse -q --verify refs/heads/%s > /dev/null" % remote_candidate, shell=True) != 0:
    remote = remote_candidate + "/"
```

First off, it's a little bit more targeted, and second, note that I flipped the condition, which matches the Git standard behavior, which is to match local branches first.

Reply:
Thank you. `rev-parse` and `for-each-ref` are two commands that I really need to learn...

Reply:
But wait, this doesn't work. Locally running it:
```
$ git rev-parse --verify refs/heads/lluiscampos
fatal: Needed a single revision
```
So maybe what you intend is to pass the rest of the string, so that if the input is `lluiscampos/feature/fancy` we call `git rev-parse ... refs/heads/feature/fancy`?

Reply:
Okey, so looking at it with fresh eyes I understood what you meant. You can ignore my previous comment; you would expect that to fail, of course.

But your logic still wasn't 100% correct, as branches with `/ ` in their name will also fail this check and then append the first part it as a false remote.

Anyway, I fixed it in an slight different way. Pleas see my last force-push.

Reply:
Of course, doh! Good that you fixed my broken example! :smile: